### ISSUE TITLE
use a constructor?

### ISSUE THREAD

**Issue by yoshuawuyts** [YerkoPalma/choo-stt#2]
Just noticed the constructor API is different from `choo-tts`. I think it'd be nice if they were similar, e.g.

```js
app.use(require('choo-stt'))   // old
app.use(require('choo-stt')()) // new
```
Thanks!

**Comment by YerkoPalma** [comment]
Yeah for sure. 
I'm planning this for a 1.0.0 stable release, if I have to do a roadmap it wold be

- Constructor like API
- Command support (see #1)

Not sure if I'm missing anything else.

**Comment by YerkoPalma** [comment]
Solved in version `1.0.0` :tada:

**Comment by yoshuawuyts** [comment]
woot! :D
